test/src/Tests/Responses/Expected/get_products.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "id": @integer@,
+        "name": "Star-Wars T-shirt",
+        "sku": "SWTS",
+        "price": 5500,
+        "sizes": @array@
+    },
+    {
+        "id": @integer@,
+        "name": "Han Solo Mug",
+        "sku": "HSM",
+        "price": 500,
+        "sizes": @array@
```

Comment:
If just for testing patterns, I suggest adding random id to the products. Then we are closer to real use-case.